test/quantization/test_quantize_fx.py

```diff
@@ -3092,28 +3094,145 @@ def test_instance_norm(self):
                 quantized_module, torch.ops.quantized.instance_norm,
                 skip_op_arg_for_functional=True)
 
-    def test_silu(self):
+    def _test_default_node_quant_handler_ops(self, module, functional, qconfig, is_reference=True, node_list=[], additional_quant_pattern_dict={}):
         class M(torch.nn.Module):
-            def __init__(self):
+            def __init__(self, mod, func):
                 super().__init__()
-                self.silu = torch.nn.SiLU()
+                self.module = mod()
+                self.functional = func
 
             def forward(self, x):
-                x = self.silu(x)
-                x = torch.nn.functional.silu(x)
+                x = self.module(x)
+                x = self.functional(x)
                 return x
 
-        data = (torch.randn((2, 2, 2, 2), dtype=torch.float),)
+        data = torch.randn((2, 2, 2, 2))
         quant_type = QuantType.STATIC
-        qconfig_dict = {
-            "": float16_static_qconfig
-        }
-        node_occurrence = {
-            ns.call_method("to"): 3
-        }
-        m = self.checkGraphModeFxOp(
-            M(), data, quant_type, custom_qconfig_dict=qconfig_dict,
-            expected_node_occurrence=node_occurrence)
+        custom_qconfig_dict = {"additional_quant_pattern": additional_quant_pattern_dict}
```

Comment:
nit: this is called `prepare_custom_config_dict` in the API definition, which is different from `qconfig_dict`.  Might be worth renaming this var, so it's consistent with the API definition, and this technically isn't a qconfig_dict.